src/components/PatchActionButtons/index.tsx

```diff
@@ -2,3 +2,4 @@ export { SchedulePatchTasks } from "./SchedulePatchTasks";
 export { UnschedulePatchTasks } from "./UnschedulePatchTasks";
 export { RestartPatch } from "./RestartPatch";
 export { SetPatchPriority } from "./SetPatchPriority";
+export { AddNotification } from "./AddNotification";
```

Comment:
I know this isnt specific to this pr but i noticed we export files in two different ways.

Like this 
`export { someModule } from "./SomeFile";`
and 
```
import { someModule }from "./SomeFile";
export { someModule };
```
We should settle on one method so its uniform through out the app.

Reply:
i added this topic to our end of sprint round table

Reply:
any of the import and then export are just leftover from when we were writing that. i'm pretty sure we tacitly decided to use `export {} from "module"`. @khelif96 you can do a PR to refactor everything to be the same if you'd like